source/AsyncRgbLedAnalyzerSettings.cpp

```diff
@@ -1,63 +1,180 @@
 #include "AsyncRgbLedAnalyzerSettings.h"
 #include <AnalyzerHelpers.h>
 
```

Comment:
add `#include <cassert>` for gcc build.

Reply:
Ah drat, Clang / libc++ are more permissive here.